gptables/examples/penguins_theme.py

```diff
@@ -0,0 +1,60 @@
+"""
+Penguins - Theme Example
+----------------------
+
+This example demonstrates how to use a custom theme in the production of a workbook.
+
+Summary statistics from the penguins dataset are used to build a ``gptables.GPTable``
+object. Elements of metadata are provided to the corresponding parameters of the class.
+Where you wish to provide no metadata in required parameters, use ``None``.
+
+The theme parameter must take either a directory or a yaml file in the ``gptables.write_workbook`` function. 
+The yaml file used in this example can be found in the themes folder as ''penguins_test_theme.yaml''.
```

Comment:
Is the theme used for testing? If not, let's swap `test` for `example` or just remove it